src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -20,8 +20,8 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the loan identified by the index number used in the displayed loan list.\n"
```

Comment:
Perhaps you can add a small notice here: "Requires a password for verification."